crossdock/client/echo.go

```diff
@@ -0,0 +1,81 @@
+package client
+
+import (
+	"crypto/rand"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"encoding/base64"
+
+	"golang.org/x/net/context"
+
+	"github.com/yarpc/yarpc-go"
+	"github.com/yarpc/yarpc-go/encoding/json"
```

Comment:
Import group ordering should be

    std library

    local packages

    third party dependencies